scopes/pipelines/builder/builder.docs.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-description: build components on isolated directories
+description: A build task manager for independent components.
```

Comment:
Component build pipelines. Customize and standardize your component build process.

Reply:
done